server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -1248,13 +1248,19 @@ func tableInvocationFromProto(p *inpb.Invocation, blobID string) (*tables.Invoca
 	i.ActionCount = p.ActionCount
 	i.BlobID = blobID
 	i.InvocationStatus = int64(p.InvocationStatus)
-	if p.ReadPermission == inpb.InvocationPermission_PUBLIC {
-		i.Perms = perms.OTHERS_READ
-	}
 	i.LastChunkId = p.LastChunkId
 	i.RedactionFlags = redact.RedactionFlagStandardRedactions
 	i.Attempt = p.Attempt
 	i.BazelExitCode = p.BazelExitCode
+
+	userGroupPerms, err := perms.GetFromContext(e.ctx, e.env)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit: I think it'd be less risky to first log a warning here to verify that we aren't getting any unexpected errors here, and have a TODO to return the error once we're more confident.

Reply:
Ack - done